tools/jdk/BUILD

```diff
@@ -193,7 +193,7 @@ genrule(
     outs = ["platformclasspath-impl.jar"],
     cmd = """
 set -eu
-TMPDIR=$$(mktemp -d)
+TMPDIR=$$(mktemp -d -t tmp.XXXXXXXX)
```

Comment:
I think this works as a cross-platform compatible `mkdtemp`

`mktemp -d 2>/dev/null || mktemp -d -t 'template'`

https://unix.stackexchange.com/questions/30091/fix-or-alternative-for-mktemp-in-os-x

Reply:
@jgavris Your suggestion works but I believe my original proposition is simpler (no need to have two commands) since explicitly specifying the template name works both on Linux and MacOSX.  Additionally, if you grep the code, you will see that my fix is more in line with the way this very same problem was handled in other parts of the code.